documentation/API_documentation/Carrier Billing-API-Readiness-Checklist.md

```diff
@@ -1,13 +1,27 @@
-# Carrier Billing API Readiness minimum criteria checklist
-
-<br>
-
-| No | Deliverables/Criteria             | Mandatory | Status (Contributed/Approved/Pending/Validated/Partly-Validated)|
-|----|-----------------------------------|-----------|------------------------|
-|  1 |API Spec                           |   Y       | Contributed            |
-|  2 |API Implementation                 |   Y       | Contributed            |
-|  3 |API Documentation                  |   Y       | Contributed            |
-|  4 |User Stories                       |   Y       | Contributed            |
-|  5 |API test cases                     |   Y       | Not Contributed        |
-|  6 |Validated by at least 2 operators  |   Y       | N/A                    |
-|  7 |Security review                    |   N       | Contributed            |
+# API Readiness Checklist
+
+Checklist for Carrier Billing v0.3.0 in r1.1.
```

Comment:
```suggestion
Checklist for Carrier Billing v0.3.0-rc.1 in r1.1.
```

(It's relevant as then the "release-candidate" column shows what should be in ... even if there is currently no difference to an initial public version)